### ISSUE TITLE
Luigi’s Mansion/Pikmin

### ISSUE THREAD

**Issue by Jtorresiii** [emukidid/swiss-gc#780]
### Describe the bug

I updated to v0.6r1420 and now I’m unable to launch Luigi’s Mansion and Pikmin with Swiss. I was able to do so with v0.6r1400 so I’ll probably go back to that version. I appreciate all of the hard work the team does for the GC community!

### Steps to reproduce

Simply launch the game on Swiss

### Expected behavior

I would expect to launch the title with no issues.

**Comment by Jtorresiii** [comment]
Operator error, working just fine.

**Comment by Extrems** [comment]
I'm not sure. I tried them using FSP at first and they worked fine, but then I moved on to an SD card and got a crash.

**Comment by Jtorresiii** [comment]
Hello, everything is running fine now, it was actually operator error on my end. I was going to start Tales of Symphonia and changed one of the options to disk read speed. Putting that setting back to to how it was before allowed me to run everything once again. Thanks!Sent from my iPhoneOn Feb 12, 2023, at 10:23 AM, Extrems ***@***.***> wrote:﻿
I'm not sure. I tried them using FSP at first and they worked fine, but then I moved on to an SD card and got a crash.
